libraries/standard/utilities/include/iot_logging_setup.h

```diff
@@ -0,0 +1,272 @@
+/*
+ * IoT Common V1.1.0
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/**
+ * @file iot_logging_setup.h
+ * @brief Defines the logging macro #IotLog.
+ */
+
+#ifndef IOT_LOGGING_SETUP_H_
+#define IOT_LOGGING_SETUP_H_
+
+/* The config header is always included first. */
+#include "config.h"
+
+/**
+ * @constantspage{logging,logging library}
+ *
+ * @section logging_constants_levels Log levels
+ * @brief Log levels for the libraries in this SDK.
+ *
+ * Each library should specify a log level by setting @ref LIBRARY_LOG_LEVEL.
+ * All log messages with a level at or below the specified level will be printed
+ * for that library.
+ *
+ * Currently, there are 4 log levels. In the order of lowest to highest, they are:
+ * - #IOT_LOG_NONE <br>
+ *   @copybrief IOT_LOG_NONE
+ * - #IOT_LOG_ERROR <br>
+ *   @copybrief IOT_LOG_ERROR
+ * - #IOT_LOG_WARN <br>
+ *   @copybrief IOT_LOG_WARN
+ * - #IOT_LOG_INFO <br>
+ *   @copybrief IOT_LOG_INFO
+ * - #IOT_LOG_DEBUG <br>
+ *   @copybrief IOT_LOG_DEBUG
+ */
+
+/**
+ * @brief No log messages.
+ *
+ * Log messages with this level will be silently discarded. When @ref
```

Comment:
There is no log message with this level. Suggested text: 

```
When @ref LIBRARY_LOG_LEVEL is #IOT_LOG_NONE, logging is disabled and no logging message are printed.
```

Reply:
Updated